src/app/+search-page/search-filters/search-filter/search-filter.component.ts

```diff
@@ -37,7 +37,7 @@ export class SearchFilterComponent implements OnInit {
    * Else, the filter should initially be collapsed
    */
   ngOnInit() {
-    this.getSelectedValues().pipe(first()).subscribe((isActive) => {
+    this.getSelectedValues().pipe(take(1)).subscribe((isActive) => {
```

Comment:
Another switch from `first` to `take(1)`.  Again, if this is needed, remove `first` from the import

Reply:
Will do